AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -38,6 +38,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 yeniasir.com.tr##.detail-subscribe
 huffingtonpost.jp##.js-newsletter-container
 deutschestheater.de##.JS-Newsletter-Registration
+ticketonline.de,eventim.de##section[data-c="eviwidget"]
```

Comment:
It will hide search a bit below:

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/166954592-c4a3fd67-76ee-40f9-b8c8-b44112e219fd.png)

</details>

```suggestion
ticketonline.de,eventim.de#?#.main-content > section[data-c="eviwidget"]:has(.evi-widget-type-general-newsletter)
```